openhands-cli/openhands_cli/user_actions/alias_setup.py

```diff
@@ -0,0 +1,333 @@
+"""Alias setup functionality for OpenHands V1 CLI.
+
+This module provides comprehensive alias setup functionality that addresses issue #10754
+by recommending uv tool install as the preferred approach over uvx for better performance
+and version control, while maintaining backward compatibility.
+"""
+
+import os
+import platform
+import subprocess
+from pathlib import Path
+from typing import Dict, List, Optional, Tuple
+
+from prompt_toolkit import HTML, print_formatted_text
+from prompt_toolkit.shortcuts import radiolist_dialog, yes_no_dialog
+
+
+class AliasSetup:
+    """Handles shell alias setup for OpenHands CLI."""
+    
+    # Shell profiles by shell type
+    SHELL_PROFILES = {
+        'bash': ['.bashrc', '.bash_profile', '.profile'],
+        'zsh': ['.zshrc', '.zprofile', '.profile'],
+        'fish': ['.config/fish/config.fish'],
+        'powershell': ['Documents/PowerShell/Microsoft.PowerShell_profile.ps1',
+                      'Documents/WindowsPowerShell/Microsoft.PowerShell_profile.ps1']
+    }
+    
+    # Alias configurations
+    ALIAS_CONFIGS = {
+        'uv_tool': {
+            'name': 'uv tool install (Recommended)',
+            'description': 'Persistent installation with better performance and version control',
+            'aliases': {
+                'openhands': 'openhands',
+                'oh': 'openhands'
```

Comment:
This alias doesn't require running `uv tool install`? does it assume that openhands cli has already been installed?